epub33/core/index.html

```diff
@@ -691,277 +732,462 @@ <h4>Conformance Checking</h4>
+						are mutually exclusive. Some resources are <a>Publication Resources</a>, some are not. Some
+						Publication Resources are allowed in the <a>spine</a> by default, while all others require
+						fallbacks. Some resources can be used in rendering <a>EPUB Content Documents</a>, while others
+						can only be used with fallbacks.</p>
 
-						<p>An <a>EPUB Publication</a> typically consists of many <a>Publication Resources</a>. These
-							resources are divided into two categories: those that do not require fallbacks (<a>Core
-								Media Type Resources</a>) and those that do (<a>Foreign Resources</a>).</p>
+					<p>Trying to understand these differences by reading the technical definitions of each category of
+						resource can be complex. To make the categorizations easier to understand, this introduction
+						introduces the concept of different axes in which the resources are used.</p>
 
-						<p>The Working Group typically only includes formats as Core Media Type Resources when they have
-							broad support in web browser cores &#8212; the rendering engines that EPUB 3 Reading Systems
-							build upon. They are an agreement between Reading System developers and EPUB Creators to
-							ensure the predictability of rendering of EPUB Publications.</p>
+					<p>The three primary resource axes are:</p>
 
-						<p>Inclusion as a Core Media Type Resource does not mean that all Reading Systems will support
-							the rendering of a resource, however. Reading Systems support also depends on the
-							capabilities of the application (e.g., a Reading System with a <a>Viewport</a> must support
-							image Core Media Type Resources, but a Reading System without a Viewport does not). Refer to
-								<a data-cite="epub-rs-33#sec-epub-rs-conf-cmt">Core Media Types</a> [[EPUB-RS-33]] for
-							more information about which Reading Systems rendering capabilities require support for
-							which Core Media Type Resources.</p>
-
-						<p>Foreign Resources come with no guarantee of rendering support, which is why they require a
-							fallback to a Core Media Type Resource. EPUB Publications should be fully consumable on any
-							conforming Reading System, so providing a fallback is necessary to ensure that the use of
-							Foreign Resources does not impact on the ability of the user to consume the content.</p>
-
-						<p>This section lists the <a href="#sec-cmt-supported">set of Core Media Type Resources</a> and
-							identifies <a href="#sec-foreign-restrictions">fallback mechanisms</a> used to satisfy the
-							consumability requirement.</p>
+					<ul>
+						<li>The <a>publication axis</a> &#8212; The publication axis describes the overarching
+							requirements for all the resources of the EPUB Publication (namely, <a>Publication
+								Resources</a> and <a>Linked Resources</a>).</li>
+						<li>The <a>spine axis</a> &#8212; The spine axis describes the content documents used in
```

Comment:
```suggestion
						<li>The <a>spine axis</a> &#8212; The spine axis describes the resources used in
```

We use the term 'resources' in all the other places, probably better here, too (although we also use the term 'document' extensively...)